xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+ --security_groups web --flavor 1 vm1
+</screen>
+</step>
+
+<step>
+      <para>
+    Assign a floating IP address to the VM. This command will also return the 
+ID of the floating IP address created:
+</para>
+<screen>
+sudo neutron floatingip-create floating
+</screen>
+</step>
+
+<step>
+<para>
+Use this command to get the ID of the port of the instance:
+</para>
+<screen>
+sudo neutron port-list
+</screen>
+</step>
+
+<step>
+<para>
+Then associate the IP address and the port, replacing the values in the 
+example with your own:
+</para>
+<screen>
+sudo neutron floatingip-associate <replaceable>192.168.1.10</replaceable> <replaceable>1234</replaceable>
+</screen>  
+  </step>
+
+  <step>
+      <para>
+Verify that you can login to your VM with SSH.
+      </para>
+  </step>
+</procedure>
+</sect1>
+
+<sect1 xml:id="sec.setupnet">
+    <title>Set Up the Second External Network</title>
+    <para>
+        Follow these steps to set up a fake external network for testing. It 
+cannot be reached by the other nodes. This setup is enough to test that, 
+at a &o_netw; level, the second public network is working correctly. You can 
+use the node itself, which is not part of the cloud, to test logging in via 
+SSH and verify that the translation from floating IP address to private IP 
+address happens correctly. You should be able to reach the VM because the 
+private IP address will be forwarded to the compute node where the 
+VM resides.
+    </para>
+    
+<procedure xml:id="pro.setupnet">  
+    <step>
+   <para>
+       First, create a VLAN. This example assigns a VLAN ID of 600:
+   </para>
+   <screen>
+sudo vconfig add eth0 600
+sudo ifconfig eth0.600 up
+sudo ethtool -K eth0.600 tx off
+   </screen>
+</step>
+
+<step>
+<para>
+    Then create a new virtual switch, and connect the new VLAN to it:
+</para>
+<screen>
+sudo ovs-vsctl add-br br-ext-1
+sudo ovs-vsctl add-port br-ext1 eth0.600
+</screen>
+</step>
+
+<step>
+<para>
+    Assign an IP address to the switch that is in the address range the second 
+external network will use, and add a route:
+</para>
+<screen>
+    sudo ip addr add <replaceable>100.100.100.1</replaceable> dev br-ext1
+sudo ip route add <replaceable>100.100.100.0/24</replaceable> dev br-ext1
+</screen>
+</step>
+
+<step>
+    <para>
+        Check the configuration of the bridge:
+    </para>
+    <screen>
+sudo ovs-vsctl show 
+</screen>
+</step>
+    </procedure>
+</sect1>   
+    
+<sect1 xml:id="sec.confignet">
+    <title>Configure the Second External Network</title>
+        <para>
+        The following steps add the networks settings, including IP address 
+pools, gateway, routing, and virtual switches. 
+</para>
+
+    <procedure xml:id="pro.confignet">
+        <step>
+            <para>
+First, stop your &chef; client, 
+then add these lines to <filename>/etc/neutron/l3_agent.ini</filename> with no 
+options:
+    </para> 
+    <screen>
+gateway_external_network_id =
+external_network_bridge =       
```

Comment:
The options are somewhat self-explanatory... still, an explanation is usually a good idea too.